src/Model/AdminManager.php

```diff
@@ -47,9 +47,20 @@ public function insert(array $comicBook): void
         $statement->execute();
     }
 
+    /**
+     * Get all authors from database.
+     */
+    public function selectAllAuthors(): array
+    {
+        $query = 'SELECT * FROM `author`;';
+
+        return $this->pdo->query($query)->fetchAll(\PDO::FETCH_ASSOC);
+    }
```

Comment:
Why didn't you create a new AuthorManager with its own selectAll() ? It's a better practice to have a manager for each table you want to manage